.sqa/config.yml

```
@@ -0,0 +1,23 @@
+  project_repos:
+    dcprepo:
+      repo: 'https://github.com/IraitzM/data-complexity-profiler'
+
+sqa_criteria:
+  qc_style:
+    repos:
+      dcprepo:
+        container: dcp-style
+        tox:
+          testenv:
+              - stylecheck
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)